src/app/Utilities.ts

```diff
@@ -11,6 +11,21 @@ const Capitalize = (string: string) => {
     }
 }
 
+/* Function to replace capitals in a string with spaces */
+const AddSpaceBeforeCaptials = (string: string) => {
```

Comment:
Yes, actually I always use capitals for functions and start variables with lower characters to keep them separated easily, this only does not apply to state setting functions because they always start with 'set' or 'get'.
So I guess functions are in PascalCase. Always weird it is called CamelCase, because camels have two humps and you would assume the first also to be in capitals, but I think 'Dromedaris Case' does not sound that good :camel:

Reply:
:smile: